### PR TITLE
Lunar Launcher | Nitter for Android | Mysterium VPN | FFShare | Feudal Tactics | Barcode Scanner | FutharkBoard

### PR DESCRIPTION
**Pull request by shuvashish76**
Extirpater best fits under Utilities category as it's a 1time thing.
From their description: _Use before selling your device, after enabling encryption, or after deleting many apps/files. Warning! Excessive use WILL destroy your device!_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -310,6 +313,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**Unexpected Keyboard**](https://github.com/Julow/Unexpected-Keyboard) <sup>**[[F-Droid](https://f-droid.org/app/juloo.keyboard2)]**</sup>
 * [**Stroke Input Method (筆畫輸入法)**](https://github.com/stroke-input/stroke-input-android) <sup>**[[F-Droid](https://f-droid.org/app/io.github.yawnoc.strokeinput)]**</sup>
 * [**nicoWnnG**](https://bitbucket-archive.softwareheritage.org/projects/da/dalb8/nicownng.html) <sup>**[[F-Droid](https://f-droid.org/app/net.gorry.android.input.nicownng)]**</sup>
+* [**FutharkBoard**](https://github.com/DrMaxNix/futharkboard) <sup>**[[F-Droid](https://www.f-droid.org/app/de.drmaxnix.futharkboard)]**</sup>
```
Comment (offa) [inline README.md:316]: The F-Droid link 404s. It's found though search, but ends in a 404 eventually.

**Review by offa — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (shuvashish76) [reply README.md:316]: Sync repo from the app or another way to check from izzy website > Select repo: F-Droid Main😅
https://apt.izzysoft.de/fdroid/index.php?repo=main

https://mastodon.technology/@IzzyOnDroid/108975473212871287

**Comment by offa** [comment]
Thanks! :-)

File: README.md (hunk 1 above)
Comment (offa) [reply README.md:316]: Link works now :+1:

**Pull request merged**